### PR TITLE
Rewrite tutorial to use vespa-cli - simplify commands and use vespa query to demonstrate query/document api MERGEOK

### PR DESCRIPTION
**Pull request by jobergum**
- Use vespa-cli to deploy app and query/document
- Some more links and query examples

I confirm that this contribution is made under the terms of the license found in the root directory of this repository's source tree and that I have the authority necessary to make this contribution on behalf of its copyright owner.

### CODE REVIEW

File: en/tutorials/news-1-getting-started.md [hunk 1]
```diff
@@ -33,24 +33,33 @@ In this part we will start with a minimal Vespa application to
...
-- python 3
-- Operating system: macOS or Linux
-- Architecture: x86_64
-- Minimum **10 GB** memory dedicated to Docker (the default is 2 GB on Macs)
+* [Docker](https://www.docker.com/) Desktop installed and running. 6GB available memory for Docker is recommended.
+  Refer to [Docker memory](https://docs.vespa.ai/en/operations/docker-containers.html#memory)
+  for details and troubleshooting
+* Operating system: Linux, macOS or Windows 10 Pro (Docker requirement)
+* Architecture: x86_64
+* Minimum **10 GB** memory dedicated to Docker (the default is 2 GB on Macs)
```
Comment (kkraune) [inline en/tutorials/news-1-getting-started.md:43]: says 6G above, probably remove the 6G as I seem to remember 10G was required for this tutorial?

File: en/tutorials/news-1-getting-started.md [hunk 2]
```diff
@@ -100,24 +106,21 @@ start it with the name `vespa`. This starts the Docker container and the
...
 Starting the container can take a short while. Before continuing, make sure
-that the configuration service is running. This is signified by a `200 OK`
-response when querying the configuration service, running on port 19071:
+that the configuration service is running. 
 
 <div class="pre-parent">
   <button class="d-icon d-duplicate pre-copy-button" onclick="copyPreContent(this)"></button>
-<pre data-test="exec" data-test-wait-for="200 OK">
-$ curl -s --head http://localhost:19071/ApplicationStatus
+<pre data-test="exec" data-test-wait-for="is ready">
```
Comment (kkraune) [inline en/tutorials/news-1-getting-started.md:113]:
```suggestion
<pre data-test="exec" data-test-assert-contains="is ready">
```

File: en/tutorials/news-3-searching.md [hunk 3]
```diff
@@ -291,14 +325,13 @@ having megabytes of text in memory per document is not cost-effective.
 
 #### Matching
 
-Another limitation is the way *matching* is done for attributes.  Consider the
-`title` field from our schema, and the document for the article with title "A
+ Consider the `title` field from our schema, and the document for the article with title "A
```
Comment (kkraune) [inline en/tutorials/news-3-searching.md:328]:
```suggestion
Consider the `title` field from our schema, and the document for the article with title "A
```

File: en/tutorials/news-5-recommendation.md [hunk 4]
```diff
@@ -509,13 +501,13 @@ restart is required so that the index can be built:
...
-<pre data-test="exec" data-test-assert-contains="prepared and activated.">
-$ (cd my-app && zip -r - .) | \
-  curl --header Content-Type:application/zip --data-binary @- \
-  localhost:19071/application/v2/tenant/default/prepareandactivate
+<pre data-test="exec" data-test-assert-contains="Success">
+$ (cd my-app && vespa deploy --wait 300) 
 </pre>
 </div>
 
+Introducing the HNSW `index` requires a restart, in this case we restart the entire system:
```
Comment (kkraune) [inline en/tutorials/news-5-recommendation.md:509]:
```suggestion
Introducing the HNSW `index` requires a content node restart, in this case we restart the entire system:
```

**Review by kkraune — APPROVED** [review]
💯

**Comment by jobergum** [comment]
Does this trigger a PR build?

**Comment by jobergum** [comment]
Odd that these link failures were not detected earlier @kkraune ? https://github.com/vespa-engine/documentation/pull/1918/commits/b6444b8c61721091b2c1f0aa296667b160cf4050

**Comment by jobergum** [comment]
Please review @kkraune :)

**Comment by kkraune** [comment]
> Odd that these link failures were not detected earlier

https://github.com/vespa-engine/documentation/pull/1929 reduces link checker concurrency, less 429 and more real 404

**Review by kkraune — APPROVED** [review]

**Pull request merged**